README.md

```diff
@@ -111,13 +94,43 @@ Amplify requires Xcode 13.4 or higher to build.
     ```swift
     import Amplify
 
-    // ... later
+    // ...
 
-    func doUpload() {
-        Amplify.Storage.uploadFile(...)
+    func signIn() async throws {
+        let signInResult = try await Amplify.Auth.signIn(...)
+        // ...
     }
     ```
 
+## Escape Hatch
+
+All services and features not listed in the [**Features/API sectios**](#featuresapis) are supported via the [Swift SDK](https://github.com/awslabs/aws-sdk-swift) or if supported by a category can be accessed via the Escape Hatch like below:
+
+```swift
+import Amplify
+import AWSS3StoragePlugin
+import AWSS3
+
+// ...
+
+guard let plugin = try Amplify.Storage.getPlugin(for: "awsS3StoragePlugin") as? AWSS3StoragePlugin else {
+    print("Unable to to cast to AWSS3StoragePlugin")
+    return
+}
+
+let awsS3 = plugin.getEscapeHatch()
+
+let accelerateConfigInput = PutBucketAccelerateConfigurationInput()
+do {
+    let accelerateConfigOutput = try await awsS3.putBucketAccelerateConfiguration(
```

Comment:
`HeadBucketOutputResponse` doesn't contain any properties, so not the most ideal example. Hence the change to an different example.